Practice Problems/01-Strings/Intermediate/strings.py

```diff
@@ -0,0 +1,47 @@
+__author__ = "John Doe"
+
+def split_string_pairs(test_string):
+	"""
+	Given a string, split it by pairs of two character. If the last character
+	is not a pair, append _ to the last one to make it a pair. Return
+	an empty list if string is empty. Examples:
+	split_string('abc') --> ['ab', 'c_']
+	split_string('abcd') --> ['ab', 'cd']
+	"""
+	pass
+
+def extract_file_name(test_string):
+	"""
+	Extract the filename from the given gibberish.
```

Comment:
Provide more information on what is the format of filename,
it should be clear that filenames in problem statement will not start with _ or number.